### PR TITLE
storage: Use rows of buttons for detail actions

### PR DESCRIPTION
**Pull request by mvollmer**
Instead of the dropdown menues.  There is enough space and the actions
are much more discoverable this way.

### CODE REVIEW

**Comment by martinpitt** [comment]
@mvollmer: Please add a screenshot for the release notes.

**Comment by mvollmer** [comment]
Looks like this now:

row-of-buttons

File: pkg/storaged/mdraid-details.jsx [hunk 1]
```diff
@@ -265,10 +265,6 @@ export class MDRaidDetails extends React.Component {
             return mdraid.RequestSyncAction("repair", {});
         }
 
-        function stop_scrub() {
-            return mdraid.RequestSyncAction("idle", {});
```
Comment (martinpitt) [inline pkg/storaged/mdraid-details.jsx]: Was it intended to entirely drop the "stop scrubbing" functionality? Couldn't this behave like the Start/Stop action, depending on whether a scrub is already running (that should be in the `SyncAction` property)? FTR, I can't test this - trying to scrub just fails with a [permission error](http://piware.de/tmp/scrub.png) for me, and this isn't exposed in gnome-disks. When I try to call it in d-feet, I get

    "Error requesting 'repair' action on RAID array '/dev/md127': Failed to set requested sync action.Failed to write 'repair' to file '/sys/class/block/md127/md/sync_action': Permission denied (2)".

This error message is confusing as the file doesn't actually exist at all. But this is at the udisks level, so not dependent on this PR.

**Comment by martinpitt** [comment]
Ah, I was trying a RAID-0, where this isn't available. For RAID-1 it does work a bit better, except that the pending job never goes away. I tested this some more, and scrubbing is broken in multiple ways for me, on current master and with this PR. I separated this into issue #8860.

**Comment by mvollmer** [comment]
I have removed the "Scrub" button here.  Let's put it back when we fix #8860.

File: pkg/storaged/mdraid-details.jsx (hunk 1 above)
Comment (mvollmer) [reply pkg/storaged/mdraid-details.jsx]: > Was it intended to entirely drop the "stop scrubbing" functionality? 

No, stopping was meant to be done by cancelling the "Repair" job that appears.  But as we found out, there are too many issues with scrubbing to have the button be this prominent now.

**Comment by mvollmer** [comment]
New screenshot:
button-row

**Comment by martinpitt** [comment]
Remaining test failures are unrelated, good enough.

**Pull request closed without merging**